zebra-rpc/qa/zallet-datadir/zallet.toml

```diff
@@ -22,8 +34,7 @@ db_path = "zaino"
 [keystore]
 identity = "identity.txt"
```

Comment:
This field [was renamed recently](https://github.com/zcash/wallet/commit/72ca81fea74a3fb9da2b7c67127716f5e95414a0#diff-f04d2344046e9b563ecb14c8778ed2a32435ccd0f8c3f6befd4f4896b2511457R472).

```suggestion
encryption_identity = "identity.txt"
```